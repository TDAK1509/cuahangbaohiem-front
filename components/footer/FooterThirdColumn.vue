<template>
  <div class="column is-hidden-mobile">
    <h4 class="has-text-weight-bold is-uppercase mb-2">Các kênh khác</h4>

    <a
      href="https://www.facebook.com/cuahangbaohiem"
      target="_blank"
      no-referrer
      class="footer-third-column__link"
    >
      <span class="icon is-medium">
        <i class="fab fa-lg fa-facebook"></i>
      </span>
    </a>
  </div>
</template>

<script>
export default {
  name: "FooterThirdColumn"
};
</script>

<style lang="scss" scoped>
.footer-third-column__link {
  color: #fff;
}
</style>
