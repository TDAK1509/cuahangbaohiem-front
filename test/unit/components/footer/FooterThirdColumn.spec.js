import { mount } from "@vue/test-utils";
import FooterThirdColumn from "@/components/footer/FooterThirdColumn.vue";

describe("FooterThirdColumn.vue", () => {
  it("matches snapshot", () => {
    const wrapper = mount(FooterThirdColumn);
    expect(wrapper.element).toMatchSnapshot();
  });
});
