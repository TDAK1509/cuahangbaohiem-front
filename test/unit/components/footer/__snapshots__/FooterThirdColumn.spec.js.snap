// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterThirdColumn.vue matches snapshot 1`] = `
<div
  class="column"
>
  <h4
    class="has-text-weight-bold is-uppercase mb-2"
  >
    
    Theo dõi chúng tôi trên mạng xã hội
  
  </h4>
   
  <a
    class="footer-third-column__link"
    href="https://www.facebook.com/cuahangbaohiem"
    no-referrer=""
    target="_blank"
  >
    <span
      class="icon is-medium"
    >
      <i
        class="fab fa-lg fa-facebook"
      />
    </span>
  </a>
</div>
`;
